docs/userGuide/diagrams/usecase.puml

```diff
@@ -3,7 +3,7 @@ left to right direction
 skinparam packageStyle rectangle
 actor customer
 actor clerk
-rectangle checkout {
+rectangle checkout as c {
```

Comment:
```suggestion
rectangle shop {
```

Perhaps it would be less confusing if we give the system a different name?

Reply:
sounds good! I have fixed it up 👍🏼